lms/views/dashboard/api/assignment.py

```diff
@@ -27,9 +27,12 @@ class AssignmentsMetricsSchema(PyramidRequestSchema):
 
     location = "querystring"
 
-    h_userids = fields.List(fields.Str())
+    h_userids = fields.List(fields.Str(), data_key="h_userid")
```

Comment:
I reckon the field should still be called with the plural in the schema and python code but using the singular form for each value in the query string.

Reply:
Yes, that makes sense to me.